.github/workflows/pr_title.yml

```
@@ -0,0 +1,109 @@
+name: PR Title Check
+
+on:
+  pull_request:
```

With `pull_request` there are no write permissions to post a comment to the PR. So this should be `pull_request_target` to enable that.
```suggestion
  pull_request_target:
```
We should however consider the consequences: https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

---

Got it.  All my testing was between branches in the same fork so I didn't notice.  So, assuming we remove the checkout and the `breaking` workflow the only user input would be the PR title and description which are used in:

```
npx commitlint --verbose <<< $COMMIT_MSG
```

I think that is a reasonable risk.